src/pages/markets/components/sub-markets/_cfds.js

```diff
@@ -42,9 +47,9 @@ const CFDs = ({ market_content, market_tab_name }) => {
                         <AvailablePlatforms dmt5 derivx />
                     )}
                 </Descriptions>
-                <StyledText font_size={'16px'} weight="bold" mt="4rem">
+                <StyledHeader as="h3" type="sub-section-title" mt="4rem">
```

Comment:
This type is deprecated and should be using the newer types instead. You shouldn't need to extend with font size as this would be set by the type.